### DISCUSSION TITLE
إقتراح منصة للتعامل مع مثل هذه الترجمات

### DISCUSSION THREAD

**Discussion by AissamYekhlef** [softvenue/i18n#70] [Ideas]
السلام عليكم أخي الكريم أما بعد فقد أعجبتني كثيرا تلك المبادرة التي قمتم بها أنت والمشاركون الذين معك في تحسين وتطوير المحتوى العربي و تعريب المجالات التقنية منه .
لذلك قمت بعمل منصة أولية للتعامل مع هذه الترجمات والحصول عليها يمكنك الإطلاع عليها من هذا الرابط والتواصل معي إن شئت لتويرها في المستقبل إن شاء الله. 
[i18n_API](https://github.com/aissamyekhlef/i18n_API/)
عصام يخلف مطور تطبيقات الويب 
شكرا

**Comment by Aissaoui-Ahmed** [comment]
<div dir=rtl>شكرا @AissamYekhlef على إقتراحك ومجهوداتك ❤️ <br>
يمكنك المساهمة في المشروع قدر ما تشاء لكن في حدود GitHub page لا اقصد أن إقتراحك لا يصلح <br>
بل محدودية Heroku، يمكنك فتح PR.<br>
أسف على البطئ في الرد<br>
تشكراتي </div>

**Comment by AissamYekhlef** [comment]
أهلا بك ء
.بالنسبة لهذا المشروع يمكننا استخدامه في عدة مجالات في المستقبل نظرا لقلة المواد المترجمة للعربية
شكرا على ردك على إقتارحي إذا كان هناك مشاريع مفتوحة المصدر يمكنني المساهمة فيها إذا احتجتم إلى مطور Laravel , PHP

**Comment by AissamYekhlef** [comment]
@Aissaoui-Ahmed إذا كان لديك أي إقتراح لعمل هكذا مشروع يسعدني لذاك.

**Comment by Aissaoui-Ahmed** [comment]
<div dir="rtl">@AissamYekhlef  يمكنك الاطلاع على #72 </div>
